tests/unit/model/changed-attrs-test.js

```diff
@@ -1275,6 +1275,91 @@ module('unit/model/changed-attrs', function(hooks) {
         }
 
         computeNestedModel(key, value) {
+          if (Array.isArray(value)) {
+            return null;
+          }
+          assert.ok(
+            !(value instanceof MegamorphicModel),
+            "We don't pass Megamorphic Models to computeNestedModel"
+          );
+          let attributesType = value && value.$type;
+          if (value !== null && typeof value === 'object') {
+            return { id: key, type: attributesType, attributes: value };
+          }
+        }
+
+        computeBaseModelName(modelName) {
+          return ['com.bookstore.projected-book', 'com.bookstore.excerpt-book'].includes(modelName)
+            ? 'com.bookstore.book'
+            : null;
+        }
```

Comment:
can we delete `computeBaseModelName` here?  Is the test actually using projections?